sns/cm/dtrebalance/sw_onwire_fop.c

```diff
@@ -96,6 +97,15 @@ m0_sns_cm_dtrebalance_sw_onwire_fop_setup(struct m0_cm *cm, struct m0_fop *fop,
 						local_ep, sw, out_interval);
 }
 
+M0_INTERNAL void m0_sns_cm_dtrebalance_wait_for_buffer(struct m0_cm *cm, struct m0_fom *fom)
```

Comment:
Can we directly use m0_sns_cm_buf_wait()? Any idea behind introducing new cm op?